src/components/RevievCard.jsx

```diff
@@ -19,23 +25,39 @@ useEffect(() => {
     })
 }, [id])
 
-    return (
-        <section className="borderElem">
-            {isLoading ? (
-          <p>Loading...</p>
-        ) : (
-            <> 
-        <h1>ReviewCard </h1>
-        {revCard && <h2>Title: {revCard.title}</h2>}
-        {revCard &&<p> : {revCard.review_body}</p>}
-
-        {revCard &&<h2>Owner : {revCard.owner}</h2>}
-        {revCard &&<h2>Id : {revCard.review_id}</h2>}
-
-        {revCard &&<h2>Category : {revCard.category}</h2>}
-        {revCard &&<img src={revCard.review_img_url} alt="Review Image" />}
-        </>
-        )}
+useEffect(() => {
+    fetchComments(id)
+        .then(({ review })  => {
+            console.log(review)
+           setComments(review)
+
+    })
+}, [id])
+
+return (
+    <section className="borderElem">
+        {isLoading ? (
+      <p>Loading...</p>
+    ) : (
+        <> 
+    <h1>ReviewCard </h1>
+    {revCard && <h2>Title: {revCard.title}</h2>}
+    {revCard &&<p> : {revCard.review_body}</p>}
+
+    {revCard &&<h2>Owner : {revCard.owner}</h2>}
+    {revCard &&<h2>Id : {revCard.review_id}</h2>}
+    {revCard &&<h2>Category : {revCard.category}</h2>}
+
+    {revCard &&<img src={revCard.review_img_url} alt="Review Image" />}
+    {revCard &&<br></br>}
+    {revCard &&<button className="show-com-button" onClick={handleShowComments}>showComments</button>}
+    {showComments &&<Comments comments={comments}/>}
```

Comment:
Keep things consistent and use a ternary operator here. It can resolve to a jsx snippet `<></>` if showComments is falsy.

Reply:
You comments should be part of a new section